src/components/plots/areaPlot.ts

```diff
@@ -36,7 +36,7 @@ export module Plot {
     }
 
     protected _getDrawer(key: string) {
-      return new Plottable._Drawer.Area(key);
```

Comment:
Maybe it'd be better to not have the interpolation mode handed in the constructor but maybe passed in through a setter.  I think the most conventional is to pass it through projections but that sounds incorrect...